plate_mapper/plate_mapper.py

```diff
@@ -68,22 +96,62 @@ def plate_mapper(input_f, barseq_f, output_f):
     barseq_f.close()
     print('  Done.')
     # Write output sequencing run file
+    samples = {}
```

Comment:
Thanks! I think I will just write the file when looping. Warning is an extra afterward function and will not stop writing the file.